core/domain/email_manager.py

```diff
@@ -370,6 +370,25 @@ def send_job_failure_email(job_id):
             mail_body.replace('\n', '<br/>'))
 
 
+def send_dummy_mail_to_admin(username):
+    """Send an email from the specified email address to admin.
+
+    Args:
+        username: str. Username of the sender.
+    """
+
+    email_name = feconf.SYSTEM_EMAIL_NAME
+    email_address = feconf.SYSTEM_EMAIL_ADDRESS
+    email_body = 'This is a test mail from %s.' % (username)
+    email_subject = 'Test Mail'
+    system_name_email = '%s <%s>' % (
+        email_name, email_address)
```

Comment:
Can probably move to previous line?

Also I think you can inline feconf.SYSTEM_EMAIL_NAME and feconf.SYSTEM_EMAIL_ADDRESS here, and get rid of the extra vars.

Reply:
Done.